datasets/lt/illegal_gambling/crawler.py

```diff
@@ -0,0 +1,35 @@
+import re 
+
+from zavod import Context
+
+def crawl_item(item, context: Context):
+
+    name = item.findtext(".//td[2]")
+    domain = item.findtext(".//td[3]")
+    contacts = item.findtext(".//td[4]")
```

Comment:
I changed the function a little to work on this specific case. What I did was:

1. Add a `skiprows` parameter to skip the first 2 rows
2. Define the headers in the parameters, I did this because the table doesn't have the `th` tags, so it wasn't identifying the name of the columns

Let me know if it looks good like this or if I should change something

Reply:
yeah the lack of th means we have to assume how many columns there are.
The colspan and rowspan also seems more tricky to parse than is justified here.

So let's use your idea of hardcoding column names, but let's also assert we match the expected column names, so we detect when they change and update our hardcoded list https://github.com/opensanctions/opensanctions/pull/636/commits/0611e60deabb93502cd7d3254383af19cde29dac